lib/msal-node/docs/caching.md

```diff
@@ -6,13 +6,17 @@ When MSAL Node acquires a token, it caches it in memory for future usage. MSAL N
 
 ```javascript
 const msal = require('@azure/msal-node');
+// Due to security reasons, secrets should not be hardcoded.
+// The dotenv npm package can be used to store secrets in a .env file (located in project's root directory)
+// that should be included in .gitignore.
```

Comment:
Can you also add "Production apps should fetch certificates from Azure KeyVault, or from other secure key vaults. See https://azure.microsoft.com/products/key-vault"